contrib/sync-upstream.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
+# - $@ are the upstream merge commits (use `git log --oneline master --merges` to show merges)
+# - Requires github-cli tool (a.k.a gh)
+# - Assumes you have an up-to-date (recently fetched) remote bitcoin-core/secp256k1
```

Comment:
Would be good to echo this comment (or something like it) if the script is called without any arguments

Reply:
Added help text if called with the wrong arguments.